docs/rfcs/README.md

```diff
@@ -0,0 +1,61 @@
+
+## Why have an RFC process?
+
+Overcoming complex technical challenges can be difficult in an open-source project where we are typically unable to meet in person to discuss solutions at depth. The RFC process is intended to provide an open venue for detailed, ongoing discussion and iteration. Further, it serves as a living record of the solution for posterity of future maintainers. The process as outlined below allows both for experimentation and focused discussion, keeping an open feedback loop and adapting to information as it becomes known.
+
+## What types of features or problems should be submitted as RFCs?
+
+While an issue can be sufficient to describe small feature requests, an RFC should be proposed for all features with impact across many areas of the code base, new patterns for extensibility APIs, and technical problems that are hard to solve without making compromises.
+
+## The RFC workflow
+
+The flow to propose a new RFC is the following:
+
+- Create an RFC document for your proposal in Markdown format.
+- Submit a pull request to the Gutenberg repository with the Markdown file included in the `docs/rfc` folder ([see examples](/docs/rfcs/)).
+- Discussion and feedback should take place in the pull request amongst the original author, Gutenberg Core team members, focus leads, project contributors, and any applicable [Core component maintainers](https://make.wordpress.org/core/components/). As the author of the proposal, you are encouraged to solicit feedback publicly, including to share your proposal in [Slack channels](https://make.wordpress.org/chat/) of interested teams.
+- As a result of discussion in the pull request and at least one [weekly editor meeting](https://make.wordpress.org/meetings/), the RFC will be accepted (merged) or rejected (closed).
+
+A merged RFC should serve as a sufficient baseline for an initial implementation, and as the de-facto documentation for the feature.
+
+### RFC status
+
+An RFC may be merged as a working document. The status of a document should be noted as a [YAML front matter (metadata)](https://github.blog/2013-09-27-viewing-yaml-metadata-in-your-documents/) in the first lines of the file, as one of the following:
+
+#### `status: Draft`
+
+A "Draft" document is one which has been merged in a state sufficient for initial implementations, but which is expected to need future iterations to refine details or revise specifics which would only become apparent in the course of said implementations. The purpose of allowing for draft documents is to allow for a steady momentum of a feature proposal and to embrace quality via experimentation.
```

Comment:
Tighten up a little, for example "refine details" and "revise specifics" are saying the same thing

```suggestion
A merged "Draft" document is sufficient for initial implementation, but expected to need future revisions based on details discovered during implementation. The purpose of allowing for draft documents is to allow for a steady momentum of a feature proposal and to embrace quality via experimentation.
```